src/cli/list.py

```diff
@@ -1,5 +1,9 @@
 """list subcommand module"""
 
+import click
 
-class List:
-    pass
+
+@click.command()
+def list_mods():
```

Comment:
Make dis list(). Function name should match the primary cli command. Aliases are secondary.